clients/filesystem-fuse/src/filesystem.rs

```diff
@@ -159,8 +161,21 @@ impl FileSystemContext {
             block_size: 4 * 1024,
         }
     }
+
+    pub(crate) fn default() -> Self {
+        FileSystemContext {
+            uid: 0,
+            gid: 0,
+            default_file_perm: 0o644,
+            default_dir_perm: 0o755,
```

Comment:
Looks like we are inconsistent with the default permissions?
In config file, the defaults were `0o600` and `0o700` respectively.